spec/mongo/uri/srv_protocol_spec.rb

```diff
@@ -583,13 +637,15 @@
           it 'sets the options on a client created with the uri' do
             expect(client.options[:read]).to eq(read)
           end
+
+          include_examples "roundtrips string"
         end
       end
 
       context 'read preference max staleness option provided' do
 
         let(:options) do
-          'readPreference=Secondary&maxStalenessSeconds=120'
+          'readPreference=secondary&maxStalenessSeconds=120'
```

Comment:
Does this lowercasing behavior have a test?

Reply:
I'm going to put this back as well